src/Components/Components/src/Binding/IFormValueCascadingParameterAttribute.cs

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Components.Binding;
+
+internal interface IFormValueCascadingParameterAttribute : ICascadingParameterAttribute
+{
+}
```

Comment:
This needs to go in Microsoft.AspNetCore.Components.Web

Reply:
I put it in `M.A.Components` so that `CascadingModelBinder` knows when it can supply a value for the cascading parameter. Maybe we could rename it to `IModelBindingCascadingParameter` or something, so we're not bringing web concepts into `M.A.Components`?